src/containers/shared/components/Copy.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import copy from '../images/copy.png';
+
+interface Props {
+  text: string;
+  className: string;
+}
+
+const Copy = (props: Props) => {
```

Comment:
i don't find the name of this component super descriptive. it could mean that it's to display "copy" as in "written content". what about something like "Copyable"?